lib/formatters/default.js

```diff
@@ -17,6 +17,7 @@ class DefaultPrinter {
       quiet: options.quiet,
       updateTodo: options.updateTodo,
       verbose: options.verbose,
+      workingDir: options.workingDir,
```

Comment:
We have both `workingDir` and `workingDirectory`. Which should we prefer?

Reply:
Where do we have `workingDirectory`?

Reply:
I think rules have `this.workingDir`, thats why I used it here. But you are right, it is actually `options.workingDirectory` from the bin script parser.

Reply:
Do we need both? Can we just have one?